README.md

```diff
@@ -1,3 +1,100 @@
 # NASA SpaceApp Challenge <br> Sri Lanka 🇱🇰
 
-content goes here...
\ No newline at end of file
+Welcome to the NASA SpaceApp Challenge Sri Lanka repository! This project is a collaborative effort for the SpaceApp Challenge, an international hackathon organized by NASA, focused on developing innovative solutions for space-related challenges.
+
+## Overview
+
+This repository contains the source code and assets for the SpaceApp Challenge Sri Lanka, showcasing our team's work and progress throughout the challenge. It includes various components such as content updates, design enhancements, and functional features to support our project's goals.
+
+## Features
+
+- **Dynamic Content**: Updated information and engaging content across various components.
+- **Starry Background Effect**: Visual enhancements with a subtle starry background to create an immersive experience.
+- **Responsive Design**: Refined layout and design elements for a consistent and polished user interface.
+- **Interactive Components**: Features including home page, about us, challenges, competition details, contact form, FAQ, hall of fame, and team profiles.
+
+## Installation
+
+To run this project locally, follow these steps:
+
+1. **Clone the Repository:**
+
+   ```bash
+   git clone https://github.com/your-username/nasa-spaceapp-challenge-lk.git
+   ```
+
+2. **Navigate to the Project Directory:**
+
+   ```bash
+   cd nasa-spaceapp-challenge-lk
+   ```
+
+3. **Install Dependencies:**
+
+   Ensure you have [Node.js](https://nodejs.org/) installed. Then, run:
+
+   ```bash
+   npm install
+   ```
+
+4. **Run the Development Server:**
+
+   ```bash
+   npm run serve
+   ```
+
+   Open your browser and go to `http://localhost:8080` to view the project.
+
+## Usage
+
+- **Home Page**: Introduction and overview of the project.
+- **About Us**: Information about the team and project background.
+- **Challenges**: Details of the challenges and solutions developed.
+- **Competition**: Information about the competition and submission details.
+- **Contact Us**: Contact form for inquiries and feedback.
+- **FAQ**: Frequently asked questions and answers.
+- **Hall of Fame**: Recognizing achievements and contributions.
+- **Team Profiles**: Information about team members and their roles.
+
+## Assets
+
+The project includes various assets such as images, fonts, and CSS files located in the `src/assets` directory.
+
+## Contributing
+
+We welcome contributions to enhance the project. Please follow these steps to contribute:
+
+1. **Fork the Repository**
+2. **Create a New Branch**:
+
+   ```bash
+   git checkout -b feature/your-feature
+   ```
+
```

Comment:
Lets put these in a `CONTRIBUTING.md` file
Sample: https://github.com/jessesquires/.github/blob/main/CONTRIBUTING.md
Ref: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors